pages/api/extension/trigger.ts

```diff
@@ -0,0 +1,48 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { getToken } from 'next-auth/jwt'
+
+export default async function triggeHandler(req: NextApiRequest, res: NextApiResponse) {
+	// For cors prefetch options request
+	if (req.method == "OPTIONS") {
+		res.setHeader("Access-Control-Allow-Headers", "Content-Type, Authorization");
+		res.setHeader("Access-Control-Allow-Origin", "*");
+		res.status(200).send("Ok");
+		return;
+	}
+	// For normal requests
+	console.info("[extension/triggeHandler] Getting setup repos info for ", req.body.owner);
+
+	if (req.method !== 'POST') {
+		return res.status(405).json({ error: 'Method Not Allowed', message: 'Only POST requests are allowed' });
+	}
+    const user = await getToken({ req: req }).catch((err) => {
```

Comment:
the name of the function `getToken` seems misleading to me. We are getting `user` from the function and it seems like we are getting token from the function.

Reply:
getToken is defined inside `next-auth/jwt`. Can't really change its name 🤷